packages/components/psammead-media-player/src/Amp/index.test.jsx

```diff
@@ -7,6 +7,7 @@ describe('Media Player: Amp', () => {
     'should render an amp-iframe with an amp-img nested inside',
     <Amp
       placeholderSrc="https://foo.bar/placeholder.png"
+      placeholderSrcset="http://foobar.com/images/ic/512xn/p01k6mtv.jpg"
```

Comment:
Would it be better to test this with a fuller example of `srcset`? At the moment it's very similar to `placeholderSrc` and we have improved the example in the readme.

Reply:
I don't think it matters because the value is just passed through without manipulation, it's opaque here I think.